CMakePresets.json

```diff
@@ -16,7 +16,10 @@
 				"rhs": "Windows"
 			},
 			"architecture": "Win32",
-			"toolchainFile": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake"
+            "toolchainFile": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake",
+			"cacheVariables": {
+				"VCPKG_TARGET_TRIPLET": "x86-windows-static-md"
```

Comment:
note for myself and maybe others wondering: the doc for triplets is really bad, but `static` sets `VCPKG_LIBRARY_LINKAGE` (build VCPKG libs as static libs) and `md` sets `VCPKG_CRT_LINKAGE` to dynamic (/MT flag)